inc/template-tags.php

```diff
@@ -68,9 +68,9 @@ function _s_entry_footer() {
 
 	edit_post_link(
 		sprintf(
-			/* translators: %s: Name of current post */
-			esc_html__( 'Edit %s', '_s' ),
-			the_title( '<span class="screen-reader-text">"', '"</span>', false )
+			/* translators: %s: Name of current post. Only visible to screen readers */
+			__( 'Edit <span class="screen-reader-text">%s</span>', '_s' ),
```

Comment:
Oh, then we will need to use `wp_kses()` like done here: https://github.com/Automattic/_s/blob/master/inc/template-tags.php#L65